spec/src/main/resources/META-INF/smithy/bsp/tags.smithy

```diff
@@ -27,3 +27,5 @@ apply StatusCode @docsPriority(7)
 apply BuildTargetCapabilities @tags(["basic"])
 
 apply BuildTargetTag @tags(["basic"])
+
+apply EnvironmentVariables @tags(["basic"])
```

Comment:
What is this weird red character here?  @agluszak maybe you know?

Reply:
looks like a bug in GH syntax highlighting library, that's just a normal space in the code